extensions/mailer/runtime/src/main/java/io/quarkus/mailer/MailTemplate.java

```diff
@@ -52,8 +52,24 @@ default MailTemplateInstance data(String key, Object value) {
 
         MailTemplateInstance addInlineAttachment(String name, File file, String contentType, String contentId);
 
+        /**
+         * 
+         * @param key
+         * @param value
+         * @return self
+         * @see io.quarkus.qute.TemplateInstance#data(String, Object)
+         */
```

Comment:
Why having the (empty) javadoc for these methods, while the others don't.

Reply:
It's not empty - there's the `@see io.quarkus.qute.TemplateInstance#data(String, Object)` ;-)

Reply:
Ah :-D